net/ariang/Makefile

```
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ariang
-PKG_VERSION:=1.3.2
-PKG_RELEASE:=1
+PKG_VERSION:=1.3.6
+PKG_RELEASE:=2
```

```suggestion
PKG_RELEASE:=1
```

This should be reset to `1`, see:
https://openwrt.org/docs/guide-developer/package-policies#versioning

---

fixing

---

@1715173329 something went wrong when sysncing from openwrt and I need to open an new pull #22125 22125